src/styles/blocks/header.scss

```
@@ -0,0 +1,70 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  &__img {
+    @include square(40px);
+  }
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  left: 0;
+  top: 0;
+  background: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    list-style-type: none;
+  }
+
+  &__link {
+    display: block;
+    padding: 23px 0;
+    text-decoration: none;
+    color: #060b35;
```

```suggestion
    color: $color-name;
```
you can apply here a variable